primerpedia.css

```diff
@@ -11,6 +11,24 @@ body, h2, .error {
 	text-align: center;
 }
 
+#notification {
+	position: fixed;
+	top: 0;
+	left: 0;
+
+	width: 100%;
+
```

Comment:
Why the empty lines? I don't think it's warranted here, considering the formatting used in the rest of the file.